101-bundle-json.md

```diff
@@ -287,6 +293,7 @@ The following fields are informational pieces of metadata designed to convey add
 
 - `description`: A short description of a bundle (OPTIONAL)
 - `keywords`: A list of keywords (OPTIONAL)
+- `labels`: Key/value pairs of type string that specify identifying attributes of the bundle.
```

Comment:
Perhaps for consistency, we append `(OPTIONAL)` to this line as well.

Reply:
So you are saying `(OPTIONAL)` isn't optional, eh? 💁‍♀️